index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>Personal Portfolio Webpage</title>
+  </head>
+  <body>
+    <nav id="navbar">
+      <ul>
+        <li>
+          <a href="#welcome-section">Projects</a>
+        </li>
+      </ul>
+    </nav>
+    <section id="welcome-section">
+      <h1>Welcome to my portfolio</h1>
+    </section>
+
+    <a
+      rel="noopener"
+      id="profile-link"
+      href="https://github.com/omarbabou"
+      target="_blank"
+    ></a>
+
+    <section id="projects">
+      <div class="project-tile">
+        <a href="https://github.com/omarbabou/Survey-Form-Html-and-Css-"></a>
+      </div>
+    </section>
+
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <script src="js/index.js"></script>
```

Comment:
Please remove the black spaces and the commented code so that it appears as clean code.

Reply:
Nice job 👍